std/outbuffer.d

```diff
@@ -385,8 +385,14 @@ class OutBuffer
     buf.write("hello");
     buf.write(cast(byte) 0x20);
     buf.write("world");
-    buf.printf(" %d", 62665);
-    assert(cmp(buf.toString(), "hello world 62665") == 0);
+    version(AArch64)
+    { // disable until va_list can be properly used
+    }
+    else
+    {
+        buf.printf(" %d", 62665);
```

Comment:
The latest version of ldc-developers/ldc#2802 gets this module to cross-compile again, but this line segfaults in the test runner in `vsnprintf`. I don't mind tests asserting, but a segfault means you can't run the rest of the tests with one invocation of the test runner, so disable this for AArch64 until we get `va_{list,arg}` working.